### ISSUE TITLE
Classify cell with scGPT

### ISSUE THREAD

**Issue by SalvatoreRa** [bowang-lab/scGPT#25]
Hi,

wonderful work, I was wondering if could be used to classify cells in inference without the need for finetuning. 

second, could be possible to identify the most important genes involved in the classification? 

Thank you very much

**Comment by subercui** [comment]
Thanks for the great questions.
1. Currently, in the preprint, we trained a new MLP classifier appended to scGPT in the fine-tuning stage. This is the "technical" reason why it is difficult to have "zero-shot" classifier since there are no such classifier layers in the pretraining stage. On the other hand, I think if you are doing something like KNN-based classification, the model can be used without fine-tuning. By KNN, I mean you used some cell embeddings of the pre-trained models with reference annotations. For a new cell, search for the nearest neighbors and assign the cell type labels by majority voting. We actually plan to set up a notebook to do something like this.

2. I think that's an interesting question! An immediate idea I have is to look at the attention weights.
